VotingApplication/VotingApplication.Web/Views/Routes/EditOptionDialog.cshtml

```diff
@@ -0,0 +1,15 @@
+﻿<div class="account-login dialog-content centered">
+    <form name="editOptionForm" ng-submit="updateOption()">
+        <h2>Edit Option</h2>
+        <div class="form-section-block centered">
+            <label for="name">Name</label>
+            <input autofocus class="form-input" id="name" ng-model="name">
+
+            <label for="description">Description</label>
+            <input class="form-input" id="description" ng-model="description">
+        </div>
+        <button class="active-btn" ng-disabled="!name || name.length === 0" ng-click="updateOption()" type="submit">Update</button>
+        <button class="inactive-btn" ng-click="dismiss()">Cancel</button>
```

Comment:
No other dialogue box on the site has a 'cancel' button. Clicking outside the box will dismiss the action, as will the built in X button. Is it really needed here?